app/controllers/audience_controller.rb

```diff
@@ -25,26 +40,17 @@ def export
     head :ok
   end
 
-  def data_by_date
-    authorize :audience, :index?
-
-    data = CreatorAnalytics::Following.new(current_seller).by_date(start_date: @start_date.to_date, end_date: @end_date.to_date)
-
-    render json: data
-  end
-
```

Comment:
API is now removed and part of partial visit on `#index` now. 

I don't think this is used on mobile app or anywhere so I think it's safe to remove, but I'm not 100% sure so lmk